pkg/neobench/parser.go

```diff
@@ -34,12 +42,25 @@ func Parse(filename, script string, weight float64) (Script, error) {
 		return Script{}, c.err
 	}
 
-	return Script{
-		Name:     filename,
-		Readonly: false, // TODO
-		Commands: commands,
-		Weight:   weight,
-	}, nil
+	output.Commands = commands
+	return output, nil
+}
+
+func metaMetaCommand(output Script, c *parseContext) *Script {
+	expect(c, '#')
+	cmd := ident(c)
+
+	switch cmd {
+	case "autocommit":
+		output.Autocommit = true
```

Comment:
WDYT of having `autocommit` be a command line flag rather than a workload config? I guess there are arguments in either direction; I imagined you might want to try "how does this query behave with autocommit vs tx function", swapping with a command line option. OTOH, "I run my workload with autocommit, and workload scripts should be self-contained descriptions of workloads" is a valid argument.

Either way: I don't like metaMetaCommand, what about introducing a dedicated meta command like `option` or `setopt` or something instead? That would also allow resolving the argument I'm having with myself above: you could build that out to say "workloads can define options for how neobench should run them, but you can - if you want - override those options on the command line.

Eg. you'd have:

```
$ cat myworkload.script

\opt autocommit true
\set myParam "banana"

MATCH (n) WHERE n.name = $param RETURN n
```

And then, in the future, we could add (gsutil style) a way to override those `setopt` calls like `neobench -o autocommit=false`

Reply:
sounds good to me

Reply:
I've implemented it so it's 
```
\ opt autocommit 
```